pyne/ionization_calculator.py

```diff
@@ -0,0 +1,598 @@
+#Begin the method to calculate ionionization energies in eV for various atoms and electronic states
+
+from pyne import nucname
+import numpy as np
+
+def function(element,charge):
```

Comment:
This function doesn't do anything because there is nothing beneath it that is indented.